src/util/versionstore.cpp

```diff
@@ -114,24 +110,24 @@ QString VersionStore::platform() {
 
 // static
 QString VersionStore::gitBranch() {
-    return kGitBranch;
+    return GitInfoStore::branch();
 }
 
 // static
 QString VersionStore::gitDescribe() {
-    return kGitDescribe;
+    return GitInfoStore::describe();
 }
 
 // static
 QString VersionStore::gitVersion() {
-    QString gitVersion = VersionStore::gitDescribe();
+    QString gitVersion = GitInfoStore::describe();
     if (gitVersion.isEmpty()) {
         gitVersion = QStringLiteral("unknown");
     }
 
     QString gitBranch = VersionStore::gitBranch();
     if (!gitBranch.isEmpty()) {
-        gitVersion.append(QStringLiteral(" (") + gitBranch + QStringLiteral(" branch)"));
+        gitVersion.append(QStringLiteral(" (") + gitBranch + QChar(')'));
```

Comment:
I'm okay with this change to make the about window a bit less wide.

Reply:
we can even remove the branch name if it matches the version

Reply:
We may, but I don't think it's necessary. In any case this should happen in the cpp code, not in the CMake code.

Reply:
ah you already did.